src/views/List.jsx

```diff
@@ -1,16 +1,37 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+
+	const handleChange = (e) => {
+		setSearchItem(e.target.value);
+	};
+
+	const filteredItems = data.filter((item) => {
+		return item.name.toLowerCase().includes(searchItem.toLowerCase());
+	});
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<label htmlFor="item-search"> Search for an item: </label>
+			<input
+				id="item-search"
+				type="search"
```

Comment:
This attribute provides you with a `clear` button in the search field. However, the clear button doesn't meet the accessibility standards required in the acceptance criteria. If you try to test the functionality as it stands, you'll notice that you can't select it via keyboard navigation. Additionally, the feature is browser-dependent, meaning that it may not appear on all browsers and may not be picked up by screen-readers. I would suggest adding a clear button next to the search input. For example:

{searchItem && (
    <button 
    type='button'
    name='clearInput'
    onClick={() => setSearchItem('')} 
    >
    X
    </button>
)}

Another alternative is to use a [semantic icon](https://semantic-ui.com/elements/icon.html#:~:text=Icons%20serve%20a%20very%20similar,for%20its%20standard%20icon%20set.)